Pod/Classes/PagingMenuController.swift

```diff
@@ -469,7 +469,7 @@ public class PagingMenuController: UIViewController, UIScrollViewDelegate {
     
     private func currentPagingViewPosition() -> PagingViewPosition {
         let pageWidth = contentScrollView.frame.width
-        let order = Int(ceil((contentScrollView.contentOffset.x - pageWidth / 2) / pageWidth))
+        let order = pageWidth <= 0 ? 0 : Int(ceil((contentScrollView.contentOffset.x - pageWidth / 2) / pageWidth))
```

Comment:
Can you use `guard` to check zero? It should be checked before calculating order. 
`guard` is supposed to be for early exit.

```swift
guard pageWidth > 0 else { return 0 }
let order = Int(ceil((contentScrollView.contentOffset.x - pageWidth / 2) / pageWidth))
```